packages/block-editor/src/components/block-variation-picker/index.js

```diff
@@ -64,8 +64,7 @@ function BlockVariationPicker( {
 			{ allowSkip && (
 				<div className="block-editor-block-variation-picker__skip">
 					<Button
-						// TODO: Switch to `true` (40px size) if possible
-						__next40pxDefaultSize={ false }
+						__next40pxDefaultSize
```

Comment:
![variation-skip-button-before](https://github.com/user-attachments/assets/d337e411-b0dc-4aa8-9a4a-dda3623da6cc)

- `__next40pxDefaultSize` has no effect here because the variation is set to `link` for this button, which overrides the size with `height: auto`.